prow/crier/controller.go

```diff
@@ -159,14 +163,14 @@ func (r *reconciler) reconcile(ctx context.Context, log *logrus.Entry, req recon
 	for _, pjob := range pjs {
 		if err := criercommonlib.UpdateReportStateWithRetries(ctx, pjob, log, r.pjclientset, r.reporter.GetName()); err != nil {
 			log.WithError(err).Error("Failed to update report state on prowjob")
-			// The error above is alreay logged, so it would be duplicated
+			// The error above is already logged, so it would be duplicated
 			// effort to combine all errors to return, only capture the last
 			// error should be sufficient.
 			lastErr = err
 		}
 	}
 
-	if pj.Status.CompletionTime != nil {
+	if pj.Complete() {
```

Comment:
This is not adjusting the rate limit. Please explain why you need to do this in the commit message.

Reply:
Reverted the change.

Reply:
> Reverted the change.

OK, but also for the future it would help reviewers if you explain why you did this in the first place (this is what I asked for...), just in case the reviewers (including myself) are missing some context. That sort of transparency in communication would help to create better PRs in general, certainly.